bin/monit-dashboard.py

```diff
@@ -9,9 +9,11 @@
 import datetime
 from collections import OrderedDict
 from operator import itemgetter
+import utils
 
 urls = ('/', 'index',
-        '/help', 'help'
+        '/help', 'help',
+        '/download', 'download'
```

Comment:
My suggestion is to update `templates/layout.html` in order to include the link there